apps/search-poc/src/utils/morph.ts

```diff
@@ -0,0 +1,21 @@
+
+import swcParser from 'swcmorphologyparser';
+
+
+const CONVERTER_URL = 'http://morph-service.ocp.bbp.epfl.ch/converter/api';
+
+export function convertToSwc(blob: any): Promise<string> {
+  const formData = new FormData();
+  formData.append('output_extension', '.swc');
+  formData.append('isBlob', 'true');
+  formData.append('file', new File([blob], 'file.asc'));
+
+  return fetch(CONVERTER_URL, { method: 'POST', body: formData })
+    .then(res => res.text())
```

Comment:
can you catch the error here (just in case). I guess you can `throw new Error('MorpServiceException')` or something

Reply:
And this we decided to leave as it is